docs/commands.md

```diff
@@ -39,7 +39,7 @@ Note: it is not recommended to run f2s on a mixture of both multi-FAST5 and sing
 *  `--to format_type`:  
    Specifies the format of output files. `format_type` can be `slow5` for SLOW5 ASCII or `blow5` for SLOW5 binary (BLOW5) [default value: blow5].
 *  `-c, --compress compression_type`:  
-   Specifies the compression method used for BLOW5 output. `compression_type` can be `none` for uncompressed binary or `zlib` for zlib-based compression [default value: zlib]. Assumes `--to blow5`.
+   Specifies the compression method used for BLOW5 output, i.e., SLOW5 output is uncompressed. `compression_type` can be `none` for uncompressed binary or `zlib` for zlib-based compression [default value: zlib]. Assumes `--to blow5`.
```

Comment:
Specifies the compression method used for BLOW5 output. `compression_type` can be `none` for uncompressed binary or `zlib` for zlib-based compression [default value: zlib]. This option is valid only if `--to blow5` is specified.